packages/nextjs/test/config/loaders.test.ts

```diff
@@ -52,42 +52,16 @@ describe('webpack loaders', () => {
         test: /sentry\.server\.config\.(jsx?|tsx?)/,
         use: [
           {
-            loader: expect.stringEndingWith('prefixLoader.js'),
-            options: expect.objectContaining({ templatePrefix: 'serverRewriteFrames' }),
-          },
-        ],
-      });
-    });
-
-    it('adds release prefix loader to server config', async () => {
-      const finalWebpackConfig = await materializeFinalWebpackConfig({
-        exportedNextConfig,
-        incomingWebpackConfig: serverWebpackConfig,
-        incomingWebpackBuildContext: serverBuildContext,
-        userSentryWebpackPluginConfig: userSentryWebpackPluginConfig,
-      });
-
-      expect(finalWebpackConfig.module.rules).toContainEqual({
-        test: /sentry\.(server|client)\.config\.(jsx?|tsx?)/,
-        use: [
-          {
-            loader: expect.stringEndingWith('prefixLoader.js'),
-            options: {
-              templatePrefix: 'release',
-              replacements: [
-                ['__RELEASE__', 'doGsaREgReaT'],
-                ['__ORG__', 'squirrelChasers'],
-                ['__PROJECT__', 'simulator'],
-              ],
-            },
+            loader: expect.stringEndingWith('valueInjectionLoader.js'),
+            options: expect.objectContaining({ values: expect.objectContaining({}) }),
```

Comment:
L/M: I'm torn on adding this here, because then maybe we should add it below, too, and that feels a little heavy-handed, given that it's a decently long comment. But I think it'd be nice to have it _somewhere_. Thoughts?

```suggestion
            // We use `expect.objectContaining({})` rather than `expect.any(Object)` to match any plain object because
            // the latter will also match arrays, regexes, dates, sets, etc. - anything whose `typeof` value is
            // `'object'`. See https://github.com/facebook/jest/blob/41bf2300895a2c00d0525d21260f0a392819871f/packages/expect/src/asymmetricMatchers.ts#L127-L129.
            options: expect.objectContaining({ values: expect.objectContaining({}) }),
```

Reply:
Good explanation. I added the comment without the link - I think future readers will trust us with that.